suite-native/app/src/navigation/AppTabNavigator.tsx

```diff
@@ -5,7 +5,7 @@ import { HomeStackNavigator } from '@suite-native/module-home';
 import { AccountsStackNavigator } from '@suite-native/module-accounts-management';
 import { SettingsStackNavigator } from '@suite-native/module-settings';
 import { AppTabsParamList, AppTabsRoutes, TabBar } from '@suite-native/navigation';
-import { useHandleDeviceRequestsPassphrase } from '@suite-native/device';
+import { useHandleDeviceRequestsPassphrase } from '@suite-native/passphrase';
```

Comment:
Does it make sense to have the hook here? Why not to put it into a RootStack? I'm asking because it is still unsure if the send will be in the TabNavigator. Very likely it will be, but let's see.

Reply:
I'd keep it for now and we can move it if there's need for it. For now, it seems that you only need it in TabNavigator

Reply:
ok, let's keep it for now, can be refactored later.